File: Lib/Telerik/Examples/QuickStart/GridView/CustomViews/NoteElement.vb
Imports System.Text
Imports Telerik.WinControls.UI

Namespace Telerik.Examples.WinControls.GridView.CustomViews
	Public Class NoteElement
		Inherits LightVisualElement
		Protected Overrides Sub InitializeFields()
			MyBase.InitializeFields()
			Me.StretchHorizontally = True
			Me.StretchVertically = True
			Me.ForeColor = Color.Black
			Me.TextAlignment = ContentAlignment.TopLeft
			Me.TextWrap = True
			Me.DisableHTMLRendering = False
			Me.ClipDrawing = True
			Me.BypassLayoutPolicies = True
		End Sub

		Protected Overrides Overloads Function MeasureOverride(ByVal availableSize As SizeF) As SizeF
			MyBase.MeasureOverride(availableSize)
			Return New SizeF(1, 1)
		End Function
	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/GridView/CustomViews/NoteElement.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls.UI;
using System.Drawing;

namespace Telerik.Examples.WinControls.GridView.CustomViews
{
    public class NoteElement: LightVisualElement
    {
        protected override void InitializeFields()
        {
            base.InitializeFields();
            this.StretchHorizontally = true;
            this.StretchVertically = true;
            this.ForeColor = Color.Black;
            this.TextAlignment = ContentAlignment.TopLeft;
            this.TextWrap = true;
            this.DisableHTMLRendering = false;
            this.ClipDrawing = true;
            this.BypassLayoutPolicies = true;
        }

        protected override SizeF MeasureOverride(SizeF availableSize)
        {
            base.MeasureOverride(availableSize);
            return new SizeF(1, 1);
        }
    }
}
